airflow/utils/clis.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
```

Comment:
nit: Maybe call this module `cli` instead of `clis`? (For consistency with `airflow/bin/cli.py`)

Reply:
+1. Can we rename this module? Right now, we have cli.py and clis.py and it is not immediately obvious to me, what clis.py class does?

Reply:
@astahlman @Alagappan I've borrowed convention from Java. For example, utilities class for Collection framework in Java is Collections. Is there any convention in Python? If there's no such as convention, I am good with cli.py as well.

Reply:
Changed to utils/cli.py